pandas/core/series.py

```diff
@@ -515,8 +515,9 @@ def _values(self):
         ----------- | ------------- | ------------- | ------------- | --------------- |
         Numeric     | ndarray       | ndarray       | PandasArray   | ndarray         |
         Category    | Categorical   | Categorical   | Categorical   | ndarray[int]    |
-        dt64[ns]    | ndarray[M8ns] | ndarray[M8ns] | DatetimeArray | ndarray[M8ns]   |
+        dt64[ns]    | ndarray[M8ns] | DatetimeArray | DatetimeArray | ndarray[M8ns]   |
```

Comment:
A bit above (beginning of the docstring), the sentence "This are the values as stored in the Block" is no longer adequate I think?

Reply:
The sentence I am quoting (the second line of the docstring) still needs to be updated